docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -76,123 +88,206 @@
       }
     }
 
-    @include hb-colorful {
-      img {
-        height: 100%;
-        width: 100%;
-        min-height: 60vh;
-        object-fit: cover;
+    img {
+      height: 100%;
+      width: 100%;
+      object-fit: cover;
+      min-height: 60vh;
 
+      @include hb-themes(('colorful', 'airy')) {
         @include grid-media-min('sm') {
           min-height: 55vh;
         }
       }
+
+      .hb-three-column--one-sidebar & {
+        @include hb-traditional {
+          @include grid-media-min('md') {
+            min-height: 65vh;
+          }
+        }
+      }
+
+      .hb-three-column--no-sidebar & {
+        @include hb-traditional {
+          @include grid-media-min('md') {
+            min-height: 75vh;
+          }
+        }
+      }
     }
   }
 
   &__text {
-    @include hb-colorful {
-      display: block;
-      @include hb-global-color('color', 'white');
-      position: absolute;
-      top: 50%;
-      transform: translate(-50%, -50%);
-      left: 50%;
-      @include hb-page-width;
-
-      h2 {
-        font-weight: hb-theme-font-weight(semibold);
+    display: block;
+    @include hb-global-color('color', 'white');
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    left: 50%;
+    @include hb-page-width;
+
+    @include hb-traditional {
+      text-align: center;
+    }
+
+    h2 {
+      font-weight: hb-theme-font-weight(semibold);
+      font-size: hb-calculate-rems(27px);
+      line-height: 117%;
+      margin: 0 0 hb-spacing-width('md');
+
+      @include grid-media-min('md') {
+        @include hb-heading-2;
+        margin: 0 0 hb-spacing-width();
+      }
+
+      @include grid-media-min('lg') {
+        @include hb-heading-1;
+      }
+    }
+
+    // must increase specificity of h2 to override line-height value set in hb-heading mixins
```

Comment:
hmmm then why do the styles on line 134-147 work?

Reply:
I removed the `.field-hs-hero-title` class and added the correct `line-height` value for traditional themes to all the breakpoints. Colorful and traditional use the same property values set in the `hb-heading` mixins except for `line-height`.

Reply:
that makes sense to me!!